common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -6084,6 +6091,18 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val SSL_HANDSHAKE_TIMEOUT_MS: ConfigEntry[Long] =
+    buildConf("celeborn.ssl.<module>.sslHandshakeTimeoutMs")
+      .categories("network", "ssl")
+      .version("0.6.0")
+      .doc("The timeout for the SSL handshake (in milliseconds). The default value is set to " +
+        "the current Netty default.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .checkValue(
+        p => p > 0 && p <= Int.MaxValue,
+        s"Invalid sslHandshakeTimeoutMs, must be a position number upto ${Int.MaxValue}")
+      .createWithDefaultString("10s")
```

Comment:
should this be in milliseconds since the config is specifying milliseconds?

Reply:
That should not matter - it will get converted to the right unit specified (which is ms).
10s is more logical than 10000 (10s in ms)

Reply:
makes sense

Reply:
This is in line with the `celeborn.ssl.<module>.trustStoreReloadIntervalMs` [default](https://github.com/apache/celeborn/blob/main/common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala#L6058). It will also get converted into milliseconds which is the time unit.

Reply:
(missed Mridul's comment when I replied to this)